src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1038,7 +1038,7 @@ private void InitEditor()
     private void InitEditorFresh()
     {
         // For now we only show a loading screen if auto-evo is not ready yet
-        if (!CurrentGame.GameWorld.IsAutoEvoFinished())
+        if (!CurrentGame.GameWorld.IsAutoEvoFinished() && Settings.Instance.RunAutoEvoDuringGamePlay.Value)
```

Comment:
I think the right thing would be to put `IsAutoEvoFinished(true)` here. Of course that change needs some verification that it doesn't accidentally start a new run when not intended. I can't immediately think of a condition that would cause that, but testing should be done to ensure correctness.